Directory.Build.targets

```diff
@@ -49,6 +49,8 @@
 
     <PackageReference Update="Castle.Windsor" Version="6.0.0" />
 
+	<PackageReference Update="Grpc.Core" Version="2.46.6" />
+	<PackageReference Update="Grpc.Net.Client" Version="2.71.0" />
```

Comment:
Is 2.46/2.71 really necessary, or is 2.0.0 also enough? Let the developers decide which version is necessary.